src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -93,25 +98,25 @@
     }
 
     public static List<CountryNote> getSampleCountryNotes() {
```

Comment:
need javadoc here, or just make it private? Probably same for function above

Reply:
done